gradio/components/video.py

```diff
@@ -36,7 +36,6 @@ class Video(
 ):
     """
     Creates a video component that can be used to upload/record videos (as an input) or display videos (as an output).
-    For the video to be playable in the browser it must have a compatible container and codec combination. Allowed
```

Comment:
Why was this deleted? A sed command gone wrong? 😂

Reply:
hahah yep! any line with "container" had to go